.github/workflows/tests.yml

```diff
@@ -130,9 +130,50 @@ jobs:
           name: web-app-server
           path: telematic_system/telematic_apps/web_app/server/coverage/*
           if-no-files-found: error
+  rosbag2_processing_service_coverage:
+    runs-on: ubuntu-latest
+    container:
+      image: ros:foxy
+      options: --user root
+    steps:
+      - name: Checkout ${{ github.event.repository.name }}
+        uses: actions/checkout@v3
+      - name: Load Environment Variables
+        shell: bash
+        run: |
+          while read -r line; do export "$line"; done < $GITHUB_WORKSPACE/telematic_system/historical_data_processing/rosbag2_processing_service/.env
+      - name: Install dependencies and build
+        shell: bash
+        run: |
+          apt-get update
+          apt-get -y install python3-coverage python3-pip python3-pytest python3-pytest-asyncio ros-foxy-rclpy unzip 
+          python3 -m pip install nats-py influxdb influxdb_client mcap-ros2-support pathlib
```

Comment:
Can we add this plugin here as well? pytest-env
So it should be:  python3 -m pip install nats-py influxdb influxdb_client mcap-ros2-support pathlib pytest-env